### ISSUE TITLE
Drop Ships (and ASFs) should be able to Flee/Exit Ground Map at Altitude 10

### ISSUE THREAD

**Issue by scJazz2** [MegaMek/megamek#615]
Quoting Total Warfare Pg 84
A unit that ascends when at Altitude 10
leaves the low-altitude map and should be placed in Row 1 of
the atmospheric hexes on the space map (unless prohibited from
doing so, see High-Altitude Movement, p. 79), in the specifi c hex
that corresponds to its present location. If players are not using the
space map, such a unit is assumed to have left the engagement.

Save Attached
[DS_Flee.sav.gz](https://github.com/MegaMek/megamek/files/1265470/DS_Flee.sav.gz)

**Comment by BLOODWOLF333** [comment]
I want this or else @ObviousTech  is just going to blow up my dropship again on the next combat drop scenario I have.

**Comment by mkerensky** [comment]
Do we even have functionality to link low-altitude/high-altitude/space maps together?  I thought not, though I remember Taharqa talking about it for Orbital Bombardment many moons ago.   Are you asking to have the 'go up' function trigger a 'fly off' effect when at altitude 10 in any hex, including mid-map hexes, for the time being?

**Comment by arlith** [comment]
That's exactly what he's asking for, and has asked for in a ticket on the SourceForge tracker.  Quoting that text from TW is pretty unhelpful because, as you point out, we don't have high altitude map nor do we link the maps in anyway.

**Comment by scJazz2** [comment]
Yes @mkerensky. @arlith quoted text just to show that rule exists

This started in relation to Spheroid DSs. However, as I researched the rules (quoted above) I realized that any ASF/CF/SC/DS should be able to "fly off" any turn if they have the MPs to reach "Altitude 11". In chatting with Bloodwolf last night the obvious complication becomes the ability to return to the map (if that option is on). What hex or map edge would a returning unit Deploy on?

**Comment by HammerGS** [comment]
Related https://github.com/MegaMek/megamek/issues/3921
